### ISSUE TITLE
An auto-play feature that isn't for quick resume

### ISSUE THREAD

**Issue by KingWing98** [Stremio/stremio-features#441]
**Describe the solution you'd like**
I'd like for there to be an auto-play feature for when you click an episode or movie as usually you have to click a link that you think is good but then ends up being to much for your internet speed. I would like it too automatically load up a link and auto-play without having to choose a link before it plays. I think it would be better if it auto played as soon as you clicked on the said episode or movie and if the link is too much for your WiFi speed, it auto moves to a lower speed link.

**Describe alternatives you've considered**
I have tried another app for this said suggestion and it works perfectly. If my episode starts to buffer, it changes the quality and link to ensure a smooth watch experience. I think it's a great thing that should be implemented into your app as it makes the app much more user friendly. 

**Additional context**
I think if the user wants to choose their own link they still can but only once they've loaded the said video and there could be a link option at the top right maybe

**Comment by TRtomasz** [comment]
closing as duplicate of https://github.com/Stremio/stremio-features/issues/337
